tests/replication/test_pusher_shard.py

```diff
@@ -95,7 +95,7 @@ def test_send_push_single_worker(self):
 
         self.make_worker_hs(
             "synapse.app.pusher",
-            {"start_pushers": True},
+            {"start_pushers": False},
```

Comment:
I'm a bit confused why we need to provide this? Does the test config default to starting push?

Reply:
`start_pushers` should be false in the config file, then previously it got set to true during `generic_worker.start` (which doesn't get run from here) and now gets set to true in the worker config class (which does get run)